test/index.js

```diff
@@ -269,4 +269,47 @@ function typedSignatureHashThrowsTest(opts) {
       sigUtil.typedSignatureHash(argument)
     })
   })
-}
\ No newline at end of file
+}
+
+//encryption test
+test("encrypt text with ECDH", async t => {
+  t.plan(4);
+
+  const senderPrivateKey =
+    "0x7f2e0a903b5e5fdee8458987386ca23c3f2db7d07eae02d3c2e6e5c6f17599a0";
+  const receiverPublicKey =
+    "44be5ff4da3cc349f101b0cb189887a55b05efc2ed97ab3054d2a16c19ca488685c70fa48556ef3d6d5342cd996ba4b41df4214947ca5ecf6324b9bc39fa5246";
+  const msgParams = { data: "My name is Satoshi Buterin" };
+
+  const result = await sigUtil.encrypt(
+    senderPrivateKey,
+    receiverPublicKey,
+    msgParams
+  );
+
+  t.ok(result.iv);
+  t.ok(result.ephemPublicKey);
+  t.ok(result.ciphertext);
+  t.ok(result.mac);
+});
+
+//decryption test
```

Comment:
what are your thoughts on adding a test that proves decryption fails with an invalid privateKey?

Reply:
That would be a good improvement, I agree we should have that.